index.js

```diff
@@ -209,48 +213,51 @@ function redisStore(args) {
    * @param {Function} [cb] - A callback that returns a potential error, otherwise null
    */
   self.set = function(key, value, options, cb) {
+    return new Promise(function(resolve, reject) {
```

Comment:
I think you can get a promise from promiseDependecy maybe (the idea is to use the promise that client provided on node-cache-manager)